README.md

```diff
@@ -1,4 +1,7 @@
 # PyoMyo
+
+#I have made a change here.
+
```

Comment:
it is not readable -- fix it

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Unreadable

Reply:
please make sure fix this code - readable not maintainable
Wrap immediately invoked function expressions in parentheses. eslint: wrap-iife

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Readable

Reply:
this code needs a fix

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
this code needs a fix

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
The code constructs all or part of an SQL command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended SQL command when it is sent to a downstream component.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
automation user test